tests/installation/first_boot.pm

```diff
@@ -28,8 +28,9 @@ sub run {
     }
 
     if (get_var("NOAUTOLOGIN") || get_var("IMPORT_USER_DATA")) {
-        assert_screen [qw(displaymanager emergency-shell emergency-mode)], $boot_timeout;
+        assert_screen [qw(displaymanager emergency-shell emergency-mode text-login)], $boot_timeout;
```

Comment:
you are trying the same approach as @yxususe did and it's still wrong. If there is only a text login prompt but a graphical one is expected then there are simply two alternatives:
* text login is correct, test variable DESKTOP=textmode is missing
* assertion about graphical session is correct, the installation is ending up in the wrong state, e.g. SLE modules missing